pytest_sugar.py

```diff
@@ -594,6 +588,24 @@ def _get_decoded_crashline(self, report):
 
         return crashline
 
+    def _get_lineno_from_report(self, report):
+        # Doctest failures in pytest>3.10 are stored in
+        # reprlocation_lines, a list of (ReprFileLocation, lines)
+        try:
+            location, lines = report.longrepr.reprlocation_lines[0]
+            return location.lineno
+        except AttributeError:
+            pass
+        # Doctest failure reports have lineno=None at least up to
+        # pytest==3.0.7, but it is available via longrepr object.
+        try:
+            return report.longrepr.reprlocation.lineno
+        except AttributeError:
+            lineno = report.location[1]
+            if lineno is not None:
+                lineno += 1
+        return lineno
```

Comment:
To be a bit more semantically correct (and satisfy IDEs/linters), should we indent this return statement?

Reply:
Oh sure, thanks (I intended initially to make a single return, but later changed it and forgot to deindent here).

Reply:
Done 👍